pipeline-model-definition/src/test/resources/json/errors/invalidWrapperType.json

```diff
@@ -16,7 +16,7 @@
     "isLiteral": true,
     "value": "none"
   },
-  "wrappers": {"wrappers": [  {
+  "options": {"options": [  {
```

Comment:
Clarity/legacy/ease of translation...this makes it a lot simpler for parsing and translating to/from groovy, IMO.